modules/base/manifests/ntp.pp

```diff
@@ -0,0 +1,21 @@
+class base::ntp (
+  $ntp_service = 'ntpd',
+){
+	# Configure  ntp.conf
+
+	file { 'ntp.conf':
+	  path    	=> '/etc/ntp.conf',
+	  ensure  	=> file,
+	  require 	=> Package['ntp'],
+	  source  	=> "puppet:///modules/base/ntp.conf",
+	  notify	=> Service['ntp'],
+	}
+	
+	# Start ntp service
+	service { 'ntp':
+	  name      => $ntp_service,
+	  ensure    => running,
+	  enable    => true,
+	  subscribe => File['ntp.conf'],
```

Comment:
I don't sure that we need both notify & subscribe in this case.
I mean notify could be enough